lib/rubocop/cop/rspec/predicate_matcher.rb

```diff
@@ -69,18 +69,22 @@ def message_inflected(predicate)
                  matcher_name: to_predicate_matcher(predicate.method_name))
         end
 
+        TO_PREDICATE_MATCHER_MAP = {
+          exist?:       'exist',
+          exists?:      'exist',
+          include?:     'include',
+          instance_of?: 'be_an_instance_of',
+          is_a?:        'be_a',
+          key?:         'have_key',
+          respond_to?:  'respond_to'
+        }.freeze
+        private_constant :TO_PREDICATE_MATCHER_MAP
+
         def to_predicate_matcher(name)
-          case name = name.to_s
```

Comment:
Side note, unrelated to this PR. This list is incomplete, since RSpec has support for eg start_with/end_with. They work with Arrays, too. Also cover/exist/...